File: test/sully/sully.c
#include <stdio.h>

int main(void) {

    int i = 5;
	i --;

	char filename[11];
	sprintf(filename, "Sully_%d.c", i);
	filename[10] = 0;

	FILE *file = fopen(filename, "w+");
	if (!file || i <= 0)
		return (1);

	char *code = "#include <stdio.h>%c%cint main(void) {%c%c	int i = %d;%c	i --;%c%c	char filename[11];%c	sprintf(filename, %cSully_%%d.c%c, i);%c	filename[10] = 0;%c%c	FILE *file = fopen(filename, %cw+%c);%c	if (!file || i <= 0)%c		return (1);%c%c	char *code = %c%s%c;%c	fprintf(file, code, 10, 10, 10, 10, i, 10, 10, 10, 10, 34, 34, 10, 10, 10, 34, 34, 10, 10, 10, 10, 34, code, 34, 10, 10, 10, 10);%c%c	return (0);%c}";
	fprintf(file, code, 10, 10, 10, 10, i, 10, 10, 10, 10, 34, 34, 10, 10, 10, 34, 34, 10, 10, 10, 10, 34, code, 34, 10, 10, 10, 10);

	return (0);
}

File: test/sully/sully.py
import os

if __name__ == "__main__":

    i = 5
    i -= 1

    if (i <= 0):
        exit(1)

    filename = "Sully_" + str(i) + ".py"
    execution = "python3 Sully_" + str(i) + ".py"
    print(execution)
    print(filename)
    code = "import os%c%cif __name__ == %c__main__%c:%c%c    i = %d%c    i -= 1%c%c    if (i <= 0):%c        exit(1)%c%c    filename = %cSully_%c + str(i) + %c.py%c%c    execution = %cpython3 Sully_%c + str(i) + %c.py%c%c%c    code = %c%s%c%c%c    file = open(filename, %cw%c)%c    file.write(code%%(10, 10, 34, 34, 10, 10, i, 10, 10, 10, 10, 10, 10, 34, 34, 34, 34, 10, 34, 34, 34, 34, 10, 10, 34, code, 34, 10, 10, 34, 34, 10, 10, 10))%c%c    os.system(execution)"

    file = open(filename, "w")
    file.write(code%(10, 10, 34, 34, 10, 10, i, 10, 10, 10, 10, 10, 10, 34, 34, 34, 34, 10, 34, 34, 34, 34, 10, 10, 34, code, 34, 10, 10, 34, 34, 10, 10, 10))

    os.system(execution)
